tests/conftest.py

```diff
@@ -27,15 +27,51 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+import shutil
+import tempfile
+
 import pytest
 from flask import Flask
+from flask_cli import FlaskCLI
+from invenio_accounts import InvenioAccounts
+from invenio_db import InvenioDB, db
+from invenio_jsonschemas import InvenioJSONSchemas
+from sqlalchemy_utils.functions import create_database, database_exists, \
+    drop_database
+
+from invenio_sipstore import InvenioSIPStore
 
 
 @pytest.fixture()
-def app():
+def app(request):
```

Comment:
Better to split up the fixture in small parts similar to: https://github.com/zenodo/zenodo/blob/master/tests/unit/conftest.py#L89-L132